<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconfigureDialog</class>
 <widget class="QDialog" name="ReconfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reconfigure GPIO Pins</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.0:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.1:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>97</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.2:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>137</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.3:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>177</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.4:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>217</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.5:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>257</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.6:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>297</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.7:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>337</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.8:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>377</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.9:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelGPIO10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>417</y>
     <width>85</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.10:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGPIO10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Reserved</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReconfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReconfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
